Platform/Samsung/sdm845/starqlte.dsc

```diff
@@ -0,0 +1,29 @@
+[Defines] 
+  VENDOR_NAME                    = Samsung
+  PLATFORM_NAME                  = starqlte
+  PLATFORM_GUID                  = 28f1a3bf-193a-47e3-a7b9-5a435eaab2ee
+  PLATFORM_VERSION               = 0.1
+  DSC_SPECIFICATION              = 0x00010019
+  OUTPUT_DIRECTORY               = Build/$(PLATFORM_NAME)
+  SUPPORTED_ARCHITECTURES        = AARCH64
+  BUILD_TARGETS                  = DEBUG|RELEASE
+  SKUID_IDENTIFIER               = DEFAULT
+  FLASH_DEFINITION               = Platform/Qualcomm/sdm845/sdm845.fdf
+  DEVICE_DXE_FV_COMPONENTS       = Platform/Samsung/sdm845/starqlte.fdf.inc
+
+!include Platform/Qualcomm/sdm845/sdm845.dsc
+
+[BuildOptions.common]
+  GCC:*_*_AARCH64_CC_FLAGS = -DENABLE_SIMPLE_INIT -DENABLE_LINUX_SIMPLE_MASS_STORAGE
```

Comment:
* "It looks like you haven't added a mainline kernel, which means that '- ENABLE_LINUX_SIMPLE_MASS_STORAGE' doesn't need to be added."
* I look forward to seeing its normal operation Xd
* Please also pay attention to Ufs issues

Reply:
star2qltechn also seems to have that option set
should that be removed as well?